### ISSUE TITLE
Matplotlib plot gets cropped in HTML output, but fine in Notebook

### ISSUE THREAD

**Issue by bradwood** [datapane/datapane#320]
### System Information

- OS: Linux
- Python version: 3.9.15
- Python environment: pip
- Using jupyter: true and false
- Datapane version: 0.15.4

### Bug / Issue
I generate exactly the same plots in Jupyter Notebook and Datapane, but when rendered in Datapane, the bottom half of the chart is removed.

## Chart from Notebook
![image](https://user-images.githubusercontent.com/3472045/199028471-87889270-855e-4f0e-a984-ce24c72ef486.png)

## Chart from Datapane
![image](https://user-images.githubusercontent.com/3472045/199028610-14f8cb10-1cd5-4260-8238-5081191fa0d5.png)

## Code
```python
    app = dp.App(
        dp.Page(title="Quality",
                blocks=[f"### {config.team.upper()}: Quality",
                        quality_blurb,
                        dp.Group(
                            dp.Plot(generate_velocity_by_type(data.issues),responsive=False),
                            dp.Text(str(generate_velocity_by_type.__doc__)),
                            columns=2,
                        ),
                ]
        ),
        dp.Page(title="Efficiency",
                blocks=[f"### {config.team.upper()}: Efficency",
                        efficiency_blurb,
                        dp.Group(
                            dp.Plot(generate_planned_vs_done_percentage(data.issues),responsive=False),
                            dp.Text(str(generate_planned_vs_done_percentage.__doc__)),
                            columns=2,
                        ),
                ]
        ),
        dp.Page(title="Business Value",
                blocks=[f"### {config.team.upper()}: Business Value",
                        'tbc',
                ]
        ),
        dp.Page(title="Downloads",
                blocks=[f"### {config.team.upper()}: Downloads"] + generate_downloads(data.excel_files)
        ),
        layout=dp.PageLayout.SIDE,
    )

    app.save(
        f'{config.output_dir}/{config.team}.html',
        formatting=dp.AppFormatting(
        width=dp.AppWidth.FULL
        )
    )

```

I've tried playing with the `responsive` setting to no avail, so now I'm stumped.

Any help would be much appreciated.

**Comment by mands** [comment]
Thanks for raising this - we'll look into it and get back to you asap!

**Comment by shahinrostami** [comment]
Hey @bradwood, thanks again for raising this.

I've [fixed the issue](https://github.com/datapane/datapane/pull/322), and it should be available to you in an upcoming release! I'll leave this issue open with the `release pending` label for you and others in the meantime.

The fix in action:

<img width="619" alt="Screenshot 2022-11-07 at 11 27 32" src="https://user-images.githubusercontent.com/15690380/200299379-7b8b9bad-58bb-451a-947b-93b1ce9d2b60.png">

**Comment by shahinrostami** [comment]
Re-opening this until the feature is released. It was auto-closed when merging the PR.

**Comment by bradwood** [comment]
thanks @shahinrostami !!!

**Comment by bradwood** [comment]
is there an ETA on the release for this @shahinrostami  @mands ? :pray:

**Comment by woobe** [comment]
Hi @bradwood we just released a new version `0.15.5` last Friday. Could you give it a try and let us know if it works for you?

**Comment by shahinrostami** [comment]
Hey @bradwood - this one has indeed been released, `0.15.5` like @woobe said!
